README.md

```diff
@@ -16,20 +24,23 @@ To be submitted as homework assignment HW-2b for the course Software Engineering
 
 ### Dependencies
 
-* Python-3
-[![Python-3](https://img.shields.io/badge/dependencies-python--3-lightgrey)](https://www.python.org/download/releases/3.0/)
-* Pandas Library
-[![Pandas Library](https://img.shields.io/badge/dependencies-pandas-lightgrey)](https://img.shields.io/badge/dependencies-pandas-lightgrey)
+* Python 3
+[![Python-3](https://img.shields.io/badge/Python-v3.7-blue)](https://www.python.org/download/releases/3.0/)
+* Pytest: [![Pytest](https://img.shields.io/badge/Pytest-v6.2.5-blue)](https://docs.pytest.org/en/6.2.x/)
 
 ## Help
 
-For any technical issues, contact:
-```
-Saurabh Nanda(snanda@ncsu.edu)
-```
+If you have specific queries, please use [Issues](https://github.com/SN-18/SE-Homework-2b/issues). If you would like to contribute, we suggest the following steps:
+1. Fork the repository (https://github.com/SN-18/SE-Homework-2b/)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Commit your changes (`git commit -am 'Add some fooBar'`)
+4. Push to the branch (`git push origin feature/fooBar`)
+5. Create a new Pull Request
+
+For more details, please read our [Contributing.md](https://github.com/SN-18/SE-Homework-2b/blob/main/CONTRIBUTING.md#contributing).
 
```

Comment:
Need to add a contact for technical issues, it's good practice. Not everyone should necessarily fork our repo and raise an issue, and there maybe some problems with build, version, or software used. Instead of deleting one, both methods should be mentioned.